examples/todos-flow/components/Link.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+import React from 'react'
+
+export type Props = {
+  active: boolean,
+  children?: React$Element<any>,
```

Comment:
@cr0cK yes, thanks. That declaration is weird though, unlike `Component` and `PureComponent` there's no "real" named export called `Element`. 

> I **guess** names with a $ are supposed to be private

Honestly I don't know what's the official advice, I might update the PR if there is a consensus on this